src/prometheus_metrics.rs

```
@@ -49,7 +57,7 @@ pub fn serve_prometheus(config: Option<&PrometheusConfig>, lemmy_context: LemmyC
           .route("/metrics", web::get().to(metrics))
       })
       .bind((bind, port as u16))
-      .unwrap_or_else(|_| panic!("Cannot bind to {}:{}", bind, port))
+      .unwrap_or_else(|_| panic!("Prometheus server cannot bind to {}:{}", bind, port))
```

Never swallow errors, include it in the log message.